src/ui/common/src/styles/theme/theme.tsx

```diff
@@ -85,5 +85,9 @@ export const theme = {
     DarkSuccessMain75: 'rgba(102, 187, 106, 0.75)',
     DarkSuccessMain50: 'rgba(102, 187, 106, 0.5)',
     LogoDarkBlue: '#002F5E',
+    LogoLight: '#9CFFFF', //(156,225,255)
+    NavBackgroundDark: '#002F5D', //(0,47,93)
```

Comment:
This is the color specified in the Figma design but seems to be very similar shade to `LogoDarkBlue`. Not currently in use so I can remove it.